packages/eslint-plugin/docs/rules/no-useless-template-literals.mdx

```diff
@@ -9,53 +9,15 @@ import TabItem from '@theme/TabItem';
 >
 > See **https://typescript-eslint.io/rules/no-useless-template-literals** for documentation.
 
-This rule reports template literals that can be simplified to a normal string literal.
+This rule reports template literals that contain substitution expressions (also variously referred to as embedded expressions or string interpolations) that are unnecessary and can be simplified.
 
-## Examples
+:::warning
+This rule is being renamed to [`no-useless-template-expression`](./no-useless-template-expression.mdx).
+After the creation of this rule, it was realized that the name `no-useless-template-literals` could be misleading, seeing as this rule only targets template literals with substitution expressions.
+In particular, it does _not_ aim to flag useless template literals that look like `` `this` `` and could be simplified to `"this"`.
+If you are looking for such a rule, you can configure the [`quotes`](./quotes.mdx) rule to do this.
```

Comment:
Heh, and `quotes` has the big _🔥 Deprecated_ notice that points to [@stylistic/ts/quotes](https://eslint.style/rules/ts/quotes). Let's point to that instead.

```suggestion
If you are looking for such a rule, you can configure the [`@stylistic/ts/quotes`](https://eslint.style/rules/ts/quotes) rule to do this.
```